cpp/daal/src/algorithms/pca/pca_dense_base.h

```diff
@@ -45,6 +45,8 @@ class PCADenseBase : public Kernel
     services::Status computeExplainedVariancesRatio(const data_management::NumericTable & eigenvalues,
                                                     const data_management::NumericTable & variances,
                                                     data_management::NumericTable & explained_variances_ratio);
+    services::Status computeNoiseVariances(const data_management::NumericTable & eigenvalues, const data_management::NumericTable & variances,
+                                           double & noise_variance);
```

Comment:
@Vika-F does it make sense to name the variable noiseVariance for daal like style?